context/app/static/js/components/savedLists/CreateListDialog/CreateListDialog.jsx

```diff
@@ -39,7 +44,7 @@ function CreateListDialog({ dialogIsOpen, setDialogIsOpen }) {
           <Button onClick={handleClose} color="primary">
             Cancel
           </Button>
-          <Button onClick={() => createList({ title, description })} color="primary" disabled={title.length === 0}>
+          <Button onClick={() => handleSubmit()} color="primary" disabled={title.length === 0}>
```

Comment:
```suggestion
          <Button onClick={handleSubmit} color="primary" disabled={title.length === 0}>
```
Would that be the same thing?